docs/index.asciidoc

```diff
@@ -199,6 +200,16 @@ For more info, check out the
 The maximum amount of time, in seconds, to wait while establishing a connection to Elasticsearch.
 Connect timeouts tend to occur when Elasticsearch or an intermediate proxy is overloaded with requests and has exhausted its connection pool.
 
+[id="plugins-{type}s-{plugin}-custom_headers"]
+===== `custom_headers`
+
+  * Value type is <<hash,hash>>
+  * Default value is empty
+
+Pass a set of key value pairs as the headers sent in each request to
+an elasticsearch node. The headers will be used for any kind of request
+These custom headers will be overidden by settings like `compression_level`.
```

Comment:
```suggestion
Pass a set of key value pairs as the headers sent in each request to an elasticsearch node. 
The headers will be used for any kind of request.
These custom headers will be overridden by settings such as `compression_level`.
```

Reply:
Please use asciidoctor's "[one sentence per line](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line)" recommendation, as it offers some pretty sweet benefits.